ngraph_bridge/ngraph_utils.cc

```diff
@@ -223,6 +223,9 @@ Status TensorToStream(std::ostream& ostream, const Tensor& tensor) {
     case DT_BOOL:
       TensorDataToStream<bool>(ostream, n_elements, data);
       break;
+    case DT_BFLOAT16:
+      TensorDataToStream<bool>(ostream, n_elements, data);
```

Comment:
It says <bool> in the template. copy-paste error perhaps.

Reply:
Good catch. Not sure what the corresponding data type for bfloat is.

Reply:
We can throw an error or return a bad status for now I guess

Reply:
Done